File: pub-sub-next/src/core/domain/entities/Subscriber.ts
import Email from "../VOs/Email";
import Name from "../VOs/Name";

export default class Subscriber {
    private _name: Name;
    private _email: Email;

    constructor(name: Name, email: Email){
        this._name = name;
        this._email = email;
    }

    public get name() {
        return this._name;
    }
    public get email() {
        return this._email;
    }
}

File: ms-notification/src/main/java/com/guilherme/delfino/pubsub/notification/domain/entity/Subscriber.java
package com.guilherme.delfino.pubsub.notification.domain.entity;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Subscriber {
    private String id;
    private String name;
    private String email;
}
